firecares/firestation/templates/firestation/firestation_add.html

```diff
@@ -82,6 +82,11 @@ <h4>{{ object.headquarters_address.get_row_display }}</h4>
                 {% block form %}
                 <div class="ct-u-marginBottom30 text-center" style="margin-top:-20px;">
                     {% with WIDGET_ERROR_CLASS='form-error' %}
+                    <script>
+                        function show_alert() {
```

Comment:
Can you inject a message block vs using an alert window?

Reply:
Actually, just remove the message...page should return instantly when celery is run w/o `CELERY_ALWAYS_EAGER`.